src/main/java/de/tum/in/www1/artemis/repository/StudentExamRepository.java

```diff
@@ -126,6 +126,8 @@ SELECT count(se) FROM StudentExam se
             """)
     Optional<StudentExam> findByExamIdAndUserId(@Param("examId") long examId, @Param("userId") long userId);
 
+    boolean existsByExam_CourseIdAndExamIdAndUserId(@Param("courseId") long courseId, @Param("examId") long examId, @Param("userId") long userId);
```

Comment:
Ensure this `boolean` can never be null, otherwise use `Boolean`.

Reply:
I couldn't see any indication that an `EXISTS` query can return false on [MySQL docs](https://dev.mysql.com/doc/refman/8.0/en/exists-and-not-exists-subqueries.html) so I think it is safe to use `boolean`